golang/main.go

```diff
@@ -10,39 +10,37 @@ import (
 )
 
 func main() {
-	var AuthToken = os.Getenv("MOMENTO_AUTH_TOKEN")
+	var authToken = os.Getenv("MOMENTO_AUTH_TOKEN")
```

Comment:
Some of our SDKs do and some don't. But the approach we were planning to take was to add banners to our outputs so the start and end of the example was clear. We have done this in Java and Python. I think while we are at it we should add banners.

Reply:
Actually, I am not sure because we are using log here. So lets skip for now.